lisa/tools/kdump.py

```diff
@@ -881,7 +881,7 @@ def _get_resource_disk_dump_path(self) -> str:
         try:
             mount_point = self.node.features[Disk].get_resource_disk_mount_point()
             dump_path = mount_point + "/crash"
-        except LisaException as e:
+        except (LisaException, NotImplementedError) as e:
```

Comment:
Can we either make the disk feature optional for the impacted case or go ahead and implement it?

Reply:
I didn't figure out a proper way to implement the disk feature in ready platform yet. and here it is to find resource disk on azure, there is no resource disk concept on BareMetal server. so to use the /var/crash to dump the crash file.

Reply:
It seems this usage of resource disk is intended to ensure there's enough disk space to save the kdump. We can add an extra method to get disk with enough data size in the disk feature. I recall a similar method exists, but I couldn't find it in the disk feature.